tests/views/test_change_response_status.py

```diff
@@ -331,3 +356,43 @@ def test_not_started_status_is_present_for_no_longer_required(self, mock_request
         self.assertEqual(response.status_code, 200)
         self.assertIn(b"No longer required", data)
         self.assertIn(b"Not started", data)
+
+    @requests_mock.mock()
+    def test_not_started_status_is_present_for_completed(self, mock_request):
```

Comment:
We really should try to move away from this pattern or massive mocking at the highest level as it gets messy. The functionality should in the backend which then means we can unit test the link creation combinations.

Reply:
I'll leave this in but I have moved the link creation to the backend with additional tests

Reply:
You can, but now you have done the work to test the dict being passed, you don't need to. The idea is you should only need to test the template logic